docs/HowToUseLoggingSystem.md

```diff
@@ -0,0 +1,96 @@
+* Debug level which is configuration in menuconfig allows printing each message.  
+* Low-level interface uses architecture-specific I/O which writes something to device directly such as up_putc().  
+  It should be used in low-level code only where it is inappropriate to use file descriptors because of disabling interrupt internally.  
+
+| Level        | Normal Interface | Low-level Interfaces | Config               |
+|--------------|------------------|----------------------|----------------------|
+| Error        | dbg              | lldbg                | CONFIG_DEBUG_ERROR   |
+| Warning      | wdbg             | llwdbg               | CONFIG_DEBUG_WARN    |
+| Info(Verbose)| vdbg             | llvdbg               | CONFIG_DEBUG_VERBOSE |
+
+Each function is wrapped macros for specific modules with the prefix of module.  
+For examples, fdbg is for filesystem error output, nwdbg is for network warning output.  
+Please refer to definitions in [os/include/debug.h](../os/include/debug.h).
+
+### How to enable debug output
+Menuconfig helps to enable debug output you are interested in as shown below:
+```
+cd $TIZENRT_BASEDIR
+cd os
+make menuconfig
+```
+1. Enable the debugging Feature.
+```
+Debug Option -> Enable Debug Features to y
+```
+2. Enable the debugging levels.
+```
+Enable Error Debug Output
+Enable Warning Debug Output
+Enable Informational(Verbose) Debug Output
+```
+The level options of each modules may be showing below depending on what you select here.
+
+3. Enable the debugging modules with level.  
+
+For examples:  
+ For enabling filesystem with error level and network with error and warning level,
+```
+ 1. Select levels, `Enable Error Debug Output` and `Enable Warning Debug output`
+ 2. Select sub-level for each modules, `File System Debug Output`, `File System Error Output`,
+           `Network Debug Output`, `Network Error Output` and `Network Warning Output`
+```
+
+## Logging Modules
+In general, all log output goes to stdout(fd = 1, usually console).  
+These can be routed to another interfaces or devices through configuration.
+
+There are 3 kinds of logging modules, LogM and 2 syslog devices.
+ * [LogM](../os/logm/README.md)
+ * [Syslog Devices](../os/drivers/syslog/README.txt) : Character device, RAM logging device  
+
+If both are enabled at the same time, all logs go to LogM.
+
+### How to enable LogM
+Please find it at [README](../os/logm/README.md) of LogM.
+
+### How to enable Syslog Devices
+Menuconfig helps to enable syslog devices as shown below:
+```
+cd $TIZENRT_BASEDIR
+cd os
+make menuconfig
+```
+Enable syslog devices with following step:
+```
+File Systems -> Advanced SYSLOG features to y
+```
+Enable specific device.
+ * character device
+  ```
+  System log character device to y
+  Set CONFIG_SYSLOG_DEVPATH as full path of device,CONFIG_SYSLOG_DEVPATH=/dev/ttyS1
+  ```
+ * RAM logging device
+  ```
+  Exit File System menu
+  Device Driver -> RAM log device support to y
+  ```
```

Comment:
May be it's good to add description about when to enable ram logging device and when to enable character device ?